state/runtime/precompiled/apex_bls_signs_verification_test.go

```diff
@@ -0,0 +1,150 @@
+package precompiled
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/bls"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	"github.com/0xPolygon/polygon-edge/crypto"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo/abi"
+)
+
+func Test_apexBLSSignatureVerification(t *testing.T) {
+	const (
+		validatorsCount = 8
+	)
+
+	encodeSingle := func(hash []byte, signature []byte, publicKey [4]*big.Int) []byte {
+		encoded, err := abi.Encode([]interface{}{hash, signature, publicKey},
+			apexBLSInputDataSingleABIType)
+		require.NoError(t, err)
+
+		return append([]byte{apexBLSSingleTypeByte}, encoded...)
+	}
+
+	encodeMulti := func(hash []byte, signature []byte, publicKeys [][4]*big.Int, bmp bitmap.Bitmap) []byte {
+		encoded, err := abi.Encode([]interface{}{hash, signature, publicKeys, bmp},
+			apexBLSInputDataMultiABIType)
+		require.NoError(t, err)
+
+		return append([]byte{0x76}, encoded...)
```

Comment:
define apexBLSMultiTypeByte type for 0x76, just like apexBLSSingleTypeByte

Reply:
https://github.com/Ethernal-Tech/blade-apex-bridge/pull/39/commits/c5f80ee1740e53b95f5d3b8c8d57a52499f44caa